### DISCUSSION TITLE
What are you using go-openai for?

### DISCUSSION THREAD

**Discussion by sashabaranov** [sashabaranov/go-openai#393] [Show and tell]
Hey all 👋🏻 

I'm curious, what are you using go-openai for? Would love to hear your stories, they would help us better tune the library to your needs! 🚀

**Comment by humanbeeng** [comment]
Building https://lepo.ai

Thanks for the awesome library. We appreciate it!

**Comment by Guaderxx** [comment]
I try to build an [AI voice chatbot](https://github.com/Guaderxx/ai-tutor), but [bark](https://github.com/suno-ai/bark) is currently too slow to generate voice compared to chatgpt.

**Comment by henomis** [comment]
I'm developing [Lingoose](https://github.com/henomis/lingoose) a Go framework for developing LLMs-based applications using multiple components such as prompt templates, pipelines, document loaders, vector store index interfaces, and may others. https://github.com/henomis/lingoose

**Reply by kmesiab** [reply]
Huge fan.

**Comment by ricboer0** [comment]
I'm using it in my project prismAId to support better systematic literature reviews. https://github.com/Open-and-Sustainable/prismAId

**Comment by kmesiab** [comment]
https://my-eq.com leverages this library.

🙂
